src/prog_models/models/battery_electrochem.py

```diff
@@ -164,16 +191,32 @@ class BatteryElectroChem(prognostics_model.PrognosticsModel):
             'Vo': 0,
             'Vsn': 0,
             'Vsp': 0,
-            'tb': 292.1 # in K, about 18.95 C
+            'tb': 292.1  # in K, about 18.95 C
         },
 
         'process_noise': 1e-3,
 
         # End of discharge voltage threshold
-        'VEOD': 3.0
-    })
-
-    def initialize(self, u, z):
+        'VEOD': 3.0, 
+        'VDropoff': 0.1 # Voltage above EOD after which voltage will be considered in SOC calculation
+    }
+
+    state_limits = {
+        'tb': (-273.15, inf)
```

Comment:
The state is in Kelvin, while the output is in Celsius. The state_limits only affect the states, so it seems like the minimum temperature should also be in Kelvin (e.g. 0 instead of -273.15).

Reply:
Good catch! Fixed